docs/GettingStarted.md

```diff
@@ -363,6 +364,7 @@ For a list of available integrations, and their configuration options, please re
 | Redis                    | `redis`                    | `>= 3.2`                 | `>= 3.2`                  | *[Link](#redis)*                    | *[Link](https://github.com/redis/redis-rb)*                                    |
 | Resque                   | `resque`                   | `>= 1.0`                 | `>= 1.0`                  | *[Link](#resque)*                   | *[Link](https://github.com/resque/resque)*                                     |
 | Rest Client              | `rest-client`              | `>= 1.8`                 | `>= 1.8`                  | *[Link](#rest-client)*              | *[Link](https://github.com/rest-client/rest-client)*                           |
+| RSpec                    | `rspec`.                   | `>= 3.0.0`               | `>= 3.0.                  | *[Link](#rspec)*.                   | *[Link](https://github.com/rspec/rspec)*                                       |
```

Comment:
```suggestion
| RSpec                    | `rspec`.                   | `>= 3.0.0`               | `>= 3.0.0`                  | *[Link](#rspec)*.                   | *[Link](https://github.com/rspec/rspec)*                                       |
```

Reply:
@marcotc thanks! Do you run any MarkDown validator in CI?